packages/tslint-config/test/rules/blueprint-html-components/true/test.tsx.fix

```diff
@@ -0,0 +1,10 @@
+import { Blockquote, Code, H1 } from "@blueprintjs/core";
```

Comment:
this is missing `Pre`

Reply:
those are `Pre`-existing conditions. there was no original import. but yeah it's weird will fix.